tew/Dockerfile

```diff
@@ -13,8 +13,8 @@ RUN set -ex; \
             curl \
             jq \
             make \
-            openssh-server \
```

Comment:
@dskard this is required for the fuzzbucket deploy since it calls scp with needs ssh

Reply:
sorry, i put it back.